### PR TITLE
Update deps and test deps.

### PR DESCRIPTION
**Pull request by JSAssassin**

### CODE REVIEW

**Review by dlongley — COMMENTED** [review]
Seems ok to me. Will leave explicit approval to others.

File: test/package.json [hunk 1]
```diff
@@ -10,32 +10,34 @@
...
   "dependencies": {
     "apisauce": "^1.1.1",
-    "bedrock": "^3.1.1",
+    "bedrock": "^4.1.1",
     "bedrock-account": "^5.0.0",
-    "bedrock-did-context": "^1.0.0",
-    "bedrock-edv-storage": "^4.1.0",
-    "bedrock-express": "^3.2.0",
+    "bedrock-did-context": "^2.0.0",
+    "bedrock-did-io": "^1.0.0",
```
Comment (dmitrizagidulin) [inline test/package.json:16]: Reminder (so we don't forget) to update to the latest bedrock-did-io PR.

**Review by dmitrizagidulin — APPROVED** [review]
:+1:

**Review by mattcollier — APPROVED** [review]
Depends on: 

- [ ] https://github.com/digitalbazaar/bedrock-kms/pull/41
- [ ] https://github.com/digitalbazaar/bedrock-kms-http/pull/51
- [ ] https://github.com/digitalbazaar/bedrock-profile/pull/68
- [x] https://github.com/digitalbazaar/bedrock-veres-one-context/pull/9
- [x] https://github.com/digitalbazaar/bedrock-did-io/pull/3

**Pull request merged**